### PR TITLE
Replace people by individuals and groups

### PR DESCRIPTION
**Pull request by lecoqlibre**
This is a proposal for issue #813.

### CODE REVIEW

File: pages/home.md [hunk 1]
```diff
@@ -49,10 +49,10 @@ permalink: /
...
               <b>Solid</b> is a 
               <a href="{{site.baseurl}}/TR/protocol">specification</a> 
-              that lets people store their data securely in decentralized data stores 
-              called <b>Pods</b>. Pods are like secure personal web servers for 
-              data. When data is stored in someone's Pod, they control which 
-              people and applications can access it.
+              that lets individuals and communities store their data securely in 
+              decentralized data stores called <b>Pods</b>. Pods are like secure 
+              web servers for data. When data is stored in a Pod, its owners 
+              control which people, organizations and applications can access it.
```
Comment (TallTed) [inline pages/home.md:55]:
```suggestion
              control which people, organizations, and applications can access it.
```

Comment (TallTed) [inline pages/home.md:55]:
```suggestion
              control which people, organizations, and applications can access it.
```

File: pages/home.md [hunk 2]
```diff
@@ -133,7 +132,7 @@ permalink: /
               Anyone or anything that accesses data in a Solid Pod uses a 
               unique ID, authenticated by a decentralized extension of OpenID 
               Connect. Solid's access control system uses these IDs to 
-              determine whether a person or application has access to a 
+              determine whether a person, organization or application has access to a
```
Comment (TallTed) [inline pages/home.md:135]:
```suggestion
              determine whether a person, organization, or application has access to a 
```

**Review by TallTed — CHANGES_REQUESTED** [review]
Just a couple of commas

File: pages/home.md [hunk 3]
```diff
@@ -132,7 +132,7 @@ permalink: /
               Anyone or anything that accesses data in a Solid Pod uses a 
               unique ID, authenticated by a decentralized extension of OpenID 
               Connect. Solid's access control system uses these IDs to 
-              determine whether a person, organization or application has access to a 
+              determine whether a person, organization, or application has access to a
```
[the remark this answers is not in the document]
Comment (lecoqlibre) [reply pages/home.md]: OK was not sure of this one, I included it.

**Review by TallTed — APPROVED** [review]

**Review by csarven — APPROVED** [review]

**Comment by lecoqlibre** [comment]
Thank you for your reviews, can someone merge this PR? Maybe @Vinnl?

**Review by VirginiaBalseiro — APPROVED** [review]

**Comment by VirginiaBalseiro** [comment]
@lecoqlibre if you can rebase this, I'll merge.

**Pull request merged**